packages/frontend/src/components/Explorer/VisualizationCard/ShowTotalsToggle.tsx

```diff
@@ -6,8 +6,6 @@ import { StyledSwitch } from './ShowTotalsToggle.styles';
 const ShowTotalsToggle: FC = () => {
```

Comment:
We should remove the entire `ShowTotalsToggle` component.

Reply:
Hello @ZeRego, I made the changes you have asked me to do. Let me know, if you need anything else